.github/CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 <h3>New features since last release</h3>
 
+* The quantum state of a QNode can now be returned using the ``state()`` return function.
+  [(#789)](https://github.com/XanaduAI/pennylane/pull/789)
+  
+  Consider the following QNode:
+  ```python
+  import pennylane as qml
+  from pennylane.beta.tapes import qnode
+  from pennylane.beta.queuing import state
+  
+  dev = qml.device("default.qubit", wires=3)
+  
+  @qnode(dev)
+  def qfunc(x, y):
+      qml.RZ(x, wires=0)
+      qml.CNOT(wires=[0, 1])
+      qml.RY(y, wires=1)
+      qml.CNOT(wires=[0, 2])
+      return state(range(3))
```

Comment:
Just a thought here, we could even make this implement full state tomography in future to work on hardware devices...